apps/playground/app/test-reset/page.tsx

```diff
@@ -54,7 +54,32 @@ export default function Test() {
                     <i>Italic text</i>
                     <iframe src="https://example.com" />
                     <img src="https://source.unsplash.com/random" />
+
+                    {/* All input types: */}
                     <input placeholder="Input control" />
+                    {/* <input placeholder="Input control" type="button" /> */}
```

Comment:
Is this all meant to be commented out?

Reply:
Yeah was just checking how it behaves; commented out since we are not handling some input types which might need a bit more work and decisions down the line

Reply:
I just handled the easy/obvious ones in this PR